src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -388,6 +392,7 @@ bool HWIntrinsicInfo::isFullyImplementedIsa(CORINFO_InstructionSet isa)
         case InstructionSet_SSE42_X64:
         case InstructionSet_Vector128:
         case InstructionSet_Vector256:
+        case InstructionSet_Vector512:
```

Comment:
Done

Reply:
The implication is that we have a `JitConfig.EnableIncompleteISAClass()` which allows ISAs not in this list to be tested using a debug/checked JIT but not a release JIT.

`Vector512` is really a special consideration in that it being in this list is "fine" because everything provides a software fallback.

We could not pre-emptively put `Avx512F` in this list because fallbacks do not exist.